metadata/root.yaml

```diff
@@ -1,9 +1,25 @@
-nfsserverprovisioner:
-  display_name: nfsserverprovisioner
-  description: nfsserverprovisioner
-  logo: nfsserverprovisioner/logo.svg
+---
+knative:
+  display_name: knative
+  description: knative
+  logo: knative/logo.svg
   category: other
-  overview: nfsserverprovisioner/overview.md
+  overview: knative/overview.md
+  resources:
+    links:
+      - type: resources
+        title: Documentation
+        link: https://github.com/mesosphere/charts/tree/master/staging/knative
+      - type: support
+        title: Submit a Support Request
+        link: https://d2iq.com/services-and-support
```

Comment:
do we actually provide support for KNative?

Reply:
if people use kfserving, somehow knativer serving has to be suppported. This link just point people to d2iq website